index.html

```diff
@@ -1049,7 +1043,7 @@ <h2 id="3Definitions">Definitions</h2>
 
 <!-- Maintain a fragment named "3whitePoint" to preserve incoming links to it -->
 <dfn id="3whitePoint">
-<dt>white point</dt></dfn>
+<dt>white point</dt></dfnn>
```

Comment:
Typo in "dfnn".

Reply:
Yes, a typo which I corrected in the original PR, and [you told me to undo](https://github.com/w3c/PNG-spec/pull/105#discussion_r833706094).